helm/draft-annotations-api/app-configs/draft-annotations-api_eks_publish.yaml

```diff
@@ -0,0 +1,8 @@
+# Values used for the deployed application.
```

Comment:
`question`: Shouldn't this file be named helm `draft-annotations-api_eks_publish_dev.yaml`

Reply:
It works as it is but I would rename it with the next PR for consistency.